Code/C++/integertobinarystring.cpp

```diff
@@ -0,0 +1,42 @@
+// this is a program  to change a integer to a binary string 
+// it takes integer as an input and changes it to binary string 
+
+
+
+#include<bits/stdc++.h>
+
+using namespace std;
+
+typedef long long int ll;        
+
+string decimalToBinary(ll n) 
+{      
+    string s = bitset<64> //finding the binary form of the number
+    (n).to_string(); //stl used to convert a number into string 
+      
+    //Finding the first occurance of "1" 
+    //to strip off the leading zeroes. 
+    const auto loc1 = s.find('1'); 
+      
+    if(loc1 != string::npos) 
+        return s.substr(loc1); 
```

Comment:
The whole purpose to add this code to the repository is to show the internal working of the algorithms, that are used to do this conversion. But here, the STL is used straight forward.
I suggest you eliminate these crucial STL functions so that the conversion algorithm becomes understandable to the readers.

- [ ] remove bitset<64>
- [ ] remove .to_string()
- [ ] remove .find()
- [ ] remove .substr